src/core/cff_parser.js

```diff
@@ -478,6 +478,9 @@ var CFFParser = (function CFFParserClosure() {
       if (state.callDepth > MAX_SUBR_NESTING) {
         return false;
       }
+      if (data === undefined) {
```

Comment:
I think that `if (!data) {` would be better here, and also let's move this new block to the top of the method instead (since no other checks make much sense when `data` isn't defined).